draft-demarco-oauth-status-attestations.md

```diff
@@ -754,7 +798,51 @@ To indicate that the content is a CWT-based Status Assertion:
   * Security considerations: See (#Security) of [[ this specification ]]
   * Interoperability considerations: n/a
   * Published specification: [[ this specification ]]
-  * Applications that use this media type: Applications using [[ this specification ]] for status assertion of tokens and Digital Credentials
+  * Applications that use this media type: Applications using [[ this specification ]] for issuing or presenting Status Assertions.
+  * Fragment identifier considerations: n/a
+  * Additional information:
+    * File extension(s): n/a
+    * Macintosh file type code(s): n/a
+  * Person &amp; email address to contact for further information: Giuseppe De Marco, gi.demarco@innovazione.gov.it
+  * Intended usage: COMMON
+  * Restrictions on usage: none
+  * Author: Giuseppe De Marco, gi.demarco@innovazione.gov.it
+  * Change controller: IETF
+  * Provisional registration? No
+
+To indicate that the content is a JWT-based Status Assertion Error:
+
+  * Type name: application
+  * Subtype name: status-assertion-error+jwt
```

Comment:
I wonder why the error needs to be protected... why not just use https://datatracker.ietf.org/doc/html/rfc7807 ?

Reply:
point taken, moved to https://github.com/peppelinux/draft-demarco-oauth-status-attestations/issues/57